sandbox.go

```diff
@@ -220,13 +220,11 @@ func (sb *sandbox) delete(force bool) error {
 			continue
 		}
 
-		if !force {
```

Comment:
This would be done during delete sandbox.
On the the contrary this would have be skipped during sandboxCleanup.